### PR TITLE
Allow base 4.20 (for GHC 9.10 support)

### PR DESCRIPTION
**Pull request by LaurentRDC**
Hello,

This is a simple PR to increase the upper bound on `base` to allow base 4.20 and GHC 9.10.

### CODE REVIEW

File: pqueue.cabal [hunk 1]
```diff
@@ -16,7 +16,8 @@ bug-reports:        https://github.com/lspitzner/pqueue/issues
 build-type:         Simple
 cabal-version:      >= 1.10
 tested-with:        GHC == 7.10.3, GHC == 8.0.2, GHC == 8.2.2, GHC == 8.4.4, GHC == 8.6.5, GHC == 8.8.4,
-                    GHC == 8.10.7, GHC == 9.0.2, GHC == 9.2.8, GHC == 9.4.7, GHC == 9.6.3, GHC == 9.8.1
+                    GHC == 8.10.7, GHC == 9.0.2, GHC == 9.2.8, GHC == 9.4.7, GHC == 9.6.3, GHC == 9.8.1,
```
Comment (konsumlamm) [inline pqueue.cabal:19]:
```suggestion
                    GHC == 8.10.7, GHC == 9.0.2, GHC == 9.2.8, GHC == 9.4.8, GHC == 9.6.5, GHC == 9.8.2,
```

**Review by konsumlamm — COMMENTED** [review]
Did you update the script by running `haskell-ci regenerate` (if not, see https://github.com/andreasabel/haskell-ci)? Can you please run it with the updated versions?

**Comment by LaurentRDC** [comment]
Sorry, I thought haskell-ci would generate the same code I did. Turns out there's a lot more.

Note that since haskell-ci does not support GHC < 8, I edited the cabal file to remove references to GHC 7.10.3

**Comment by LaurentRDC** [comment]
Closed in favour of #135

**Pull request closed without merging**